doc/man3/EVP_DigestVerifyInit.pod

```diff
@@ -50,6 +56,12 @@ The B<EVP> interface to digital signatures should almost always be used in
 preference to the low level interfaces. This is because the code then becomes
 transparent to the algorithm used and much more flexible.
 
+EVP_DigesVerify() is a single part operation which verifies a single block of
+data in one function. For algorithms that support streaming it is equivalent
+to calling EVP_DigestVerifyUpdate() and EVP_DigestVerifyFinal(). For
+algorithms which do not support streaming (e.g. PureEdDSA) it is the only way
```

Comment:
Nit: most style guides I look at recommend commas after (and before) "e.g." and "i.e.".